src/api/spec/controllers/person_controller_spec.rb

```diff
@@ -24,9 +24,27 @@
   end
 
   describe 'POST #post_userinfo' do
+    before do
+      login user
+    end
+
+    context 'when using default authentication' do
+      let!(:old_password) { user.password_digest }
+
+      before do
+        request.env["RAW_POST_DATA"] = "password_has_changed"
```

Comment:
Whats this for?

Reply:
To set the HTTP Body for the request. Somehow you need to transfer the password to the API. This is done by the body. See: https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/person_controller.rb#L53